### DISCUSSION TITLE
Usefulness of traffic_signals:sound quest

### DISCUSSION THREAD

**Discussion by hungerburg** [streetcomplete/StreetComplete#4186] [General]
In my home town, almost all pedestrian traffic signals can sound. But the wont do that for people, lacking a compatible transponder, to unlock the feature. It is free for blind people from their association. As a consequence, it depends on arbitrary circumstances, if crossings get tagged "traffic_signals:sound=no|yes" by StreetComplet users.

I think, this is not the least a shortcoming of the specification. The devices here are from Langmatz, a German manufacturer, the ones with built in speakers. Some posts have extra speakers mounted on top. All silent, if you not happen by chance to hear them activated by a blind person, that feels the need to have the guidance, they provide.

As the blind are the target audience of the information gathered from this quest: How does it affect usefulness?

**Comment by mnalis** [comment]
It looks like current definition of [traffic_signals:sound=*](https://wiki.openstreetmap.org/wiki/Key:traffic_signals:sound) does not support such remote-operated sound signals, although such issue has been mentioned at [talk page](https://wiki.openstreetmap.org/wiki/Talk:Key:traffic_signals:sound). It likely needs to be proposed at [tagging mailing list](https://wiki.openstreetmap.org/wiki/Tagging_mailing_list) / documented in wiki, before it could be used in StreetComplete.

**Reply by matkoniecz** [reply]
And in SC it would be likely used to skip such objects in resurvey quest (if exists for this quest) and to prevent such questions if marked already.

**Comment by timothywashere** [comment]
If this is the situation in all of Austria then the quest should probably be disabled for Austria

**Reply by hungerburg** [reply]
No idea about all of the country. In my city, its like that, everywhere residential. Only on a primary highway far from housing the signals sound all the time.

Maybe, the ones who can turn on the sound, do not need OSM to know, the ones that use OSM, are not helped by knowing?

If so, the tagging applied here is misleading in those cases even then, as it looks like some mappers happened by chance to hear the sounds.

**Reply by Helium314** [reply]
It's not the same in all of Austria, e.g. in Vienna there are sound signals without requiring any special transponder.
But since quests can only be disabled per-country, I still think it should actually be disabled for Austria (and Czechia too, as there are similar sound signals activated by such transponders).

Alternatively there could be some tag specifically for "always-on" sound signals, but that doesn't belong in SC discussion.

**Reply by matkoniecz** [reply]
Is it typical situation or present only in a tiny number of settlements?

**Reply by Helium314** [reply]
Accoding to information I found, in Vienna all sound signals emit a sound for locating the light, while in many / most other areas the situation is as @hungerburg described.

**Reply by hungerburg** [reply]
There are two sounds: the knocking one to locate the post, the howling one to find the post on the other side of the street.

From the [Talk on the tag](https://wiki.openstreetmap.org/wiki/Talk:Key:traffic_signals:sound#Remote-operated_signal_sound) I gather, that it's like that in most of France.

(In a CS comment, I asked a local mapper that recently tagged sound=yes, From the reply, I think they mistook the sound, that the vibration device makes during the green phase as this sound signal. Vibration can be activated from a button, but sound cannot, to prevent misuse. I have not found the German wording of the quest. Is it sufficiently clear?)

**Reply by robelix** [reply]
I did misunderstand that question.
"Gibt es hier akustische Signale für Sehbehinderte?" (Are there acoustic signals for the visually impaired?)  included the humming from the vibration device for me.

Maybe something more like "Gibt es hier akustische Signale die Sehbehinderten helfen den Taster zu finden?" (Are there acoustic signals to help the visually impaired find the button?)

Well, I would have answered that with no - since I just learned about the existence of these transponders.

**Comment by hungerburg** [comment]
When I asked a contributor today how he thinks about that, traffic signlas emitting sound only when a certain radio gets close by, I was told: 1) Such is not survey-able OTG for somebody not having the device, 2) It is only a small community that has the device.

My thoughts on: 1) So survey result will differ by chance, depending who is close there at the time. 2) The small community is actually why this gets mapped, so why disregard them?

Time to amend the Wiki documentation? [Talk there](https://wiki.openstreetmap.org/wiki/Talk:Key:traffic_signals:sound#Remote-operated_signal_sound) moves slow as molasses.

**Reply by matkoniecz** [reply]
> [Talk there](https://wiki.openstreetmap.org/wiki/Talk:Key:traffic_signals:sound#Remote-operated_signal_sound) moves slow as molasses.

I recommend tagging mailing list or https://community.openstreetmap.org/ if you prefer discussion happening at scale faster than decades and it is not attempt to reach specific wiki user or about specific edits
